README.adoc

```diff
@@ -3,164 +3,159 @@
 :icons: font
 :source-highlighter: prettify
 :project_id: gs-scheduling-tasks
+:build_name: gs-scheduling-tasks
+:build_version: 0.0.1-SNAPSHOT
+:build_system: gradle
 
 This guide walks you through the steps for scheduling tasks with Spring.
 
 == What You Will Build
 
 You will build an application that prints out the current time every five seconds by using
-Spring's `@Scheduled` annotation.
+Spring Framework's https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/scheduling/annotation/Scheduled.html[`@Scheduled`^] annotation.
 
+ifndef::env-exclude-spring-initializr[]
 == What You Need
 
-:java_version: 1.8
-include::https://raw.githubusercontent.com/spring-guides/getting-started-macros/main/prereq_editor_jdk_buildtools.adoc[]
+* About 15 minutes
+* A favorite text editor or IDE
+* https://www.oracle.com/java/technologies/downloads/[Java 17^] or later
 
-include::https://raw.githubusercontent.com/spring-guides/getting-started-macros/main/how_to_complete_this_guide.adoc[]
+== How to complete this guide
+Like most Spring https://spring.io/guides[Getting Started guides^], you can start from scratch and complete each step or you can jump straight to the solution, by viewing the code in https://github.com/spring-guides/gs-scheduling-tasks[this repository^]
+
+To **see the end result**, do the following:
```

Comment:
Change `do the following:` to `either` and remove the two line returns (to make one sentence).

Reply:
I made some significant changes to this section that changed the flow.